### PR TITLE
Fix 4.1.0 lint

### PR DESCRIPTION
**Pull request by samtstern**

### CODE REVIEW

**Comment by samtstern** [comment]
@SUPERCILEX now that `quality.gradle` is gone where should I declare that we want to ignore the `MissingTranslation` for now.  In the `baseline`?

File: auth/src/main/java/com/firebase/ui/auth/ui/phone/PhoneActivity.java [hunk 1]
```diff
@@ -79,7 +79,7 @@ protected void onFailure(@NonNull Exception e) {
             protected void onSuccess(@NonNull PhoneVerification verification) {
                 if (verification.isAutoVerified()) {
                     Toast.makeText(
-                            PhoneActivity.this, R.string.fui_verified, Toast.LENGTH_LONG).show();
+                            PhoneActivity.this, R.string.fui_auto_verified, Toast.LENGTH_LONG).show();
```
Comment (SUPERCILEX) [inline auth/src/main/java/com/firebase/ui/auth/ui/phone/PhoneActivity.java]: Whoops, thanks for catching that! 😊

**Comment by SUPERCILEX** [comment]
@samtstern right here: https://github.com/firebase/FirebaseUI-Android/blob/fc9e46aa15d6b6ac3664d5eb08146da4e9006aee/build.gradle.kts#L88. Or in the module's build file like here: https://github.com/firebase/FirebaseUI-Android/blob/fc9e46aa15d6b6ac3664d5eb08146da4e9006aee/auth/build.gradle.kts#L13.

**Comment by SUPERCILEX** [comment]
Looks like lint didn't like your baseline changes. Would probably just be easier to move the lambda around. 🤷‍♂️

**Comment by samtstern** [comment]
WHY DOES SQUASH AND MERGE NOT WORK

**Comment by samtstern** [comment]
image

**Comment by SUPERCILEX** [comment]
Lol, [report it](https://github.com/contact)?

**Pull request merged**